scalafix-tests/unit/src/main/scala/scalafix/test/FqnRule.scala

```diff
@@ -7,23 +7,23 @@ import scalafix._
 case class FqnRule(index: SemanticdbIndex)
     extends SemanticRule(index, "FqnRule") {
   override def fix(ctx: RuleCtx): Patch =
-    ctx.addGlobalImport(importer"scala.collection.immutable")
+    ctx.addGlobalImport(importer"scala.collection.immutable").atomic
```

Comment:
I would be careful to automatically infer atomic for import patches or rename. It's possible to imagine rules that rename AND add an import. More explicit is better IMO.

When I think of it, we should not change `.asPatch` either. Instead, I think it's better to add a helper method `.eachAtomic` (name up for debate) for `Iterable[Patch]` so that it's possible to do `tree.collect { ... }.eachAtomic.asPatch`.

Reply:
I guess we both did the review at the same time so our comments are all intertwined 😄

Reply:
I didn't quiet get this, should I make it atomic or not? If I don't then we'll getting our error `One or more rules used by test scala/test/FqnRule.scala emit non-atomic top level patches`

Reply:
I made the change